docs/content/dagster-cloud/deployment/agents/running-multiple-agents.mdx

```diff
@@ -68,10 +68,6 @@ If using the CloudFormation template provided by Dagster, the number of replicas
 
 If you want to run multiple agents in an environment where each agent can not access the others' resources (for example, multiple kubernetes namespaces or different clusters), you need to enable the `isolated_agents` option. This is supported for all agent types.
 
-<Note>
-  It is not currently possible to control which agent a request goes to.
```

Comment:
can you also remove "Submitting runs to a specific agent isn't currently supported." at the top of the page?